src/rules/no-duplicates.js

```diff
@@ -2,21 +2,198 @@ import resolve from 'eslint-module-utils/resolve'
 import docsUrl from '../docsUrl'
 
 function checkImports(imported, context) {
-  for (let [module, nodes] of imported.entries()) {
-    if (nodes.size > 1) {
-      for (let node of nodes) {
-        context.report(node, `'${module}' imported multiple times.`)
+  for (const [module, nodes] of imported.entries()) {
+    if (nodes.length > 1) {
```

Comment:
shouldn't this be `nodes.size`?
```suggestion
    if (nodes.size > 1) {
```

Reply:
I changed to an array, since it does not make sense to use a `Set` for storing AST objects – they will never deduplicate anyway. And even if they did, that's not what we want here – we need all the duplicate imports.